Schemata/sarif-external-property-file-schema-2.1.0.json

```diff
@@ -153,7 +153,7 @@
     },
 
     "addresses": {
-      "description": "Addresses that will be merged with with an external run.",
+      "description": "Addresses that will be merged with an external run.",
```

Comment:
Please create a different PR for changes on published artifacts (on docs.oasis-open.org i.e. standards work products.